app/assets/stylesheets/helpers/_button.scss

```diff
@@ -0,0 +1,15 @@
+// TODO: We can move this into GOV.UK Publishing Components if it makes sense to do so.
+
+@import "design-patterns/buttons"; // Elements Button import globally depends on mixins defined in here.
+@import "elements/helpers"; // Elements Button import globally depends on mixins defined in here.
+// DEBT/RISK: Includes a .example-highlight-grid global CSS helper
+// https://github.com/alphagov/govuk_elements/blob/874e287/public/sass/elements/_helpers.scss#L12
+// and a reimplementation of .visually-hidden
+// https://github.com/alphagov/govuk_elements/blob/874e287/public/sass/elements/_helpers.scss#L36
+@import "elements/buttons";
```

Comment:
Have these issues be raised against elements?
Can elements be easily refactored to avoid these problems?

This _feels_ heavy handed. I'd expect the styling to be ported from `frontend` where it originally lived, and a more focused switch to elements/govuk-frontend to come later.